### PR TITLE
use proper path to arm64 EFI firmware for qemu

### PR DESCRIPTION
**Pull request by mischief**

### CODE REVIEW

**Comment by mischief** [comment]
retest this please

**Comment by mischief** [comment]
maybe this should use `/build/$BOARD/usr/share/edk2-armvirt/QEMU_EFI.fd`, otherwise something needs to copy it to the image dir during build_image.

**Comment by marineam** [comment]
Yeah, reading from /build/... sounds fine.

**Comment by marineam** [comment]
If it doesn't exist already probably should add a sdk.BoardRoot() function

File: sdk/repo.go [hunk 1]
```diff
@@ -95,6 +95,10 @@ func DefaultBoard() string {
 	return string(board)
 }
 
+func BoardRoot() string {
```
Comment (marineam) [inline sdk/repo.go]: Support passing in a string and use DefaultBoard if string is blank.

File: cmd/kola/options.go [hunk 2]
```diff
@@ -27,6 +29,11 @@ var (
 		"amd64-usr": sdk.BuildRoot() + "/images/amd64-usr/latest/coreos_production_image.bin",
 		"arm64-usr": sdk.BuildRoot() + "/images/arm64-usr/latest/coreos_developer_image.bin",
 	}
+
+	kolaDefaultBIOS = map[string]string{
+		"amd64-usr": "",
```
Comment (marineam) [inline cmd/kola/options.go]: Use `bios.bin` and then this flag can be used for both. And then it is possible to do kola on x86_64 uefi too :)

**Comment by mischief** [comment]
ptal

File: platform/qemu.go [hunk 3]
```diff
@@ -33,6 +33,10 @@ type QEMUOptions struct {
 	// DiskImage is the full path to the disk image to boot in QEMU.
 	DiskImage string
 	Board     string
+
+	// BIOSImage is the full path to the BIOS image to use in QEMU.
+	// Currently only used for UEFI in aarch64.
```
Comment (marineam) [inline platform/qemu.go]: comment no longer correct

**Comment by marineam** [comment]
a comment needs fixing but otherwise LGTM

**Pull request merged**